packages/node_modules/@webex/plugin-meetings/src/meetings/util.js

```diff
@@ -22,6 +24,15 @@ MeetingsUtil.extractDestination = (destination, type) => {
   return dest;
 };
 
+// ** This is a workaround to fix Firefox and google android not downloading H264 codec on start
+MeetingsUtil.triggerH264Codec = async () => {
+  let testPeerconnection = new window.RTCPeerConnection();
+
+  await testPeerconnection.createOffer();
```

Comment:
I just had one thought here, would it make sense to test if the codec is available before creating a temporary  new peer connection and offer? I'm not sure of the penalties if any for doing so each time.

There's a `HTMLMediaElement.canPlayType()` that is supported by all browsers. Would it make sense to use it here? 

In case you want to look into it:
https://davidwalsh.name/detect-supported-video-formats-javascript
https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/canPlayType